### DISCUSSION TITLE
Youtube annoyance: annotations & endscreen

### DISCUSSION THREAD

**Discussion by gradogirl** [uBlockOrigin/uAssets#24263] [Q&A] [answered]
Hello there!

How do I suggest adding the follow filters to uBlock filters – Annoyances?

`||youtube.com/*/annotations_module.js
`
`||youtube.com/*/endscreen.js
`

Reference: https://www.reddit.com/r/uBlockOrigin/comments/tcswua/tip_noncosmetic_filters_to_remove_junk_from/

**Comment by Yuki2718** [comment, marked as answer]
Definitely not for public lists. Something annoying to you is something necessary for others.

**Reply by gradogirl** [reply]
Trying to add this to public lists for uBOL usage (MV3 in chrome). 

Not the answer I was hoping for, but this would look like a site breakage for everyone else.

Thanks Yuki2718!

**Comment by jvschiavo** [comment]
How do I add that on my local machine?
